src/simcore_service_deployment_agent/git_url_watcher.py

```diff
@@ -277,12 +279,46 @@ async def _init_repositories(repos: list[GitRepo]) -> dict:
     return description
 
 
+async def _check_if_tag_on_branch(repo_path: str, branch: str, tag: str) -> bool:
+    cmd = [
+        "git",
+        "log",
+        "--tags",
+        "--simplify-by-decoration",
+        '--pretty="format:%ai %d"',
+    ]
+    try:
+        data = await run_cmd_line(cmd, repo_path)
+    except CmdLineError as e:
+        raise RuntimeError(
+            " ".join(cmd), "The command was invalid and the cmd call failed."
+        ) from e
+    if not data:
+        return False
+    if "malformed object name" in data:
```

Comment:
This ia leftover from "git branch --contains" which I used before switching to this cmd call. (see: https://stackoverflow.com/a/5549506) In this case it would return 0 and print this error. Doesnt apply nowl will remove.